app/views/shared/_navbar.html.erb

```diff
@@ -23,7 +23,13 @@
             <%= link_to "Post Animal", new_animal_path, class: "nav-link btn btn-lightblue mx-2" %>
           </li>
           <li class="nav-item dropdown">
-            <%= image_tag "professionalpicture.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          
+            <% if current_user.photo.attached? %>
```

Comment:
render this too

Reply:
Rendered! But needed its own shared html to function as a drop down.

Reply:
We could try to refactor all these avatar_html's into a single one, but for right now they all sort of have different uses/ styling.